config/examples/Zonestar/P802M/Configuration.h

```diff
@@ -21,8 +21,6 @@
  */
 #pragma once
 
-#define CONFIG_EXAMPLES_DIR "Zonestar/P802M"
-
```

Comment:
These lines should remain since all configs have this define now.

Reply:
I'm not an git expert. For some reason the changes I made to the ENDER 3 V2 config to add BLTouch got included in this pull request as well. I did however fix the missing CONFIG_... define.
Is it possible to only approve the Zonestar changes and Ignore the Ender 3 V2 changes?

Reply:
You'll need to revert the changes you don't want to include or you can close this PR and start a clean one without the other changes.